tests/integration/actions/test_password_actions.py

```diff
@@ -0,0 +1,129 @@
+#!/usr/bin/env python3
+# Copyright 2022 Canonical Ltd.
+# See LICENSE file for licensing details.
+
+import logging
+
+import pytest
+from pytest_operator.plugin import OpsTest
+
+from tests.integration.helpers import (
+    APP_NAME,
+    MODEL_CONFIG,
+    SERIES,
+    get_admin_secrets,
+    get_application_unit_ids,
+    get_leader_unit_id,
+    get_leader_unit_ip,
+    http_request,
+    run_action,
+)
+from tests.integration.tls.test_tls import TLS_CERTIFICATES_APP_NAME
+
+logger = logging.getLogger(__name__)
+
+
+@pytest.mark.actions_tests
```

Comment:
I feel this could be added into the charm_tests tag

Reply:
I agree, if we end up with too many actions, we could always externalize those later